.drone.yml

```diff
@@ -1,7 +1,26 @@
+---
 kind: pipeline
+type: kubernetes
 name: default
+
+tolerations:
+  - effect: NoSchedule
+    key: app
+    operator: Exists
+node_selector:
+  app: drone
```

Comment:
You don't need this as it will be set by default.

Reply:
hmm, Ok will fix it in next PR